Libplanet/Blocks/InvalidBlockMinTxException.cs

```diff
@@ -0,0 +1,64 @@
+#nullable enable
+using System;
+using System.Runtime.Serialization;
+using Libplanet.Blockchain.Policies;
+
+namespace Libplanet.Blocks
+{
+    /// <summary>
+    /// The exception that is thrown when a <see cref="Block{T}"/>'s
+    /// <see cref="Block{T}.Transactions"/> Count is too small.
+    /// </summary>
+    [Serializable]
+    public sealed class InvalidBlockMinTxException : InvalidBlockException, ISerializable
```

Comment:
Would there be more descriptive name than this?  The exception thrown when there are too many transactions is `BlockExceedingTransactionsException`.  I suggest `BlockInsufficientTransactionsException`.

Also, this needs to be added to the changelog.

Reply:
Done. See the latest commit.

Reply:
Um... on the other hand, now that I think of it, the new descriptive name doesn't exactly match its use case since the exception is thrown while "gathering" transactions, not while appending. 🙃